submissions/nataliaArtsiukh/DocumentObjectModel/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Inline+One&family=Cinzel+Decorative&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <script src="./script.js"></script>
+    <title>Document</title>
+</head>
+<body>
+    <div class="all-content">
+        <img src="./img/burg.png" alt="" class="burger">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item">Howard Phillips Lovecraft</li>
+                <li class="nav-item">Azathoth</li>
+                <li class="nav-item">Yog-Sothoth</li>
+                <li class="nav-item">Shub-Niggurath</li>
+                <li class="nav-item">Nyarlathotep</li>
+                <li class="nav-item">Cthulhu</li>
```

Comment:
Generate navigation with JS to avoid editing in two places. Use one source of truth: your object with data. With this approach easy to edit page content in the future.